test/config_tests.erl

```diff
@@ -572,4 +694,46 @@ getmsg(Pid) ->
 
 
 n_handlers() ->
-    length(gen_event:which_handlers(config_event)).
+    length(handlers()).
+
+handlers() ->
+    Handlers = gen_event:which_handlers(config_event),
+    [Pid || {config_listener, {?MODULE, Pid}} <- Handlers].
+
+%% copy/paste from test_util
```

Comment:
I am not sure if we can just reuse the functions from test_util. I noticed that @davisp didn't use functions from that module. So I decided to copy/paste. Let me know if we can reuse functions from test_util.